docs/source/getting-started/welcome.rst

```diff
@@ -1,46 +1,86 @@
+   :target: https://deepchecks.com/?utm_source=docs.deepchecks.com/stable&utm_medium=referral&utm_campaign=welcome
    :alt: Deepchecks Logo
    :align: center
+   :width: 80%
 
-========================
-Welcome to Deepchecks!
-========================
+================================
+Welcome to Deepchecks Testing!
+================================
 
-Deepchecks is the leading tool for testing, validating and 
-:doc:`monitoring <deepchecks-mon:getting-started/welcome>` your machine learning models
-and data, and it enables doing so with minimal effort. Deepchecks accompanies you through
-various validation and testing needs such as verifying your data's integrity, inspecting its distributions,
-validating data splits, evaluating your model and comparing between different models.
+Deepchecks is a holistic tool for testing, validating and monitoring your machine learning models
+and data, throughout the model's lifecycle. It enables you to identify probelems with your
+data quality, distributions, and model's performance with minimal effort.
+
+See more info in the :ref:`Deepchecks Components for Continuous Validation <welcome__deepchecks_components>`
+section, along with the direct links to the documentation of each component.
 
 .. image:: /_static/images/general/checks-and-conditions.png
-   :alt: Deepchecks Suite of Checks
-   :width: 75%
+   :alt: Deepchecks Testing Suite of Checks
+   :width: 65%
    :align: center
 
 |
 
 .. _welcome__start_working:
 
-Start Working with Deepchecks Testing
-==========================================
+Get Started with Deepchecks Testing
+========================================
 
 .. grid:: 1
     :gutter: 3
    
-    .. grid-item-card:: 🤓 Concepts & Guides 🤓
-        :link-type: doc
-        :link: /general/index
+    .. grid-item-card:: 🏃‍♀️ Quickstarts 🏃‍♀️
+        :link-type: ref
+        :link: welcome__quickstarts
          
-        A comprehensive view of deepchecks concepts,
-        customizations, and core use cases.
-   
+        Downloadable end-to-end guides, demonstrating how to start testing your data & model
+        in just a few minutes.
+
+    .. grid-item-card:: 💁‍♂️ Get Help & Give Us Feedback 💁
+        :link-type: ref
+        :link: welcome__get_help
+
+        Links for how to interact with us via our `Slack Community  <https://www.deepchecks.com/slack>`__
+        or by opening `an issue on Github <https://github.com/deepchecks/deepchecks/issues>`__.
+
+
     .. grid-item-card:: 💻  Install 💻 
         :link-type: doc
         :link: /getting-started/installation
 
         Full installation guide (quick one can be found in quickstarts)
 
+    .. grid-item-card:: 🤓 General: Concepts & Guides 🤓
+        :link-type: ref
+        :link: general__index
+         
+        A comprehensive view of deepchecks concepts,
+        customizations, and core use cases.
+
+    .. grid-item-card:: 🔢 Tabular 🔢
+        :link-type: ref
+        :link: tabular__index
```

Comment:
I'm not sure I understand why here the links are tabular__index and not tabular_index, but trust that it works(:

Reply:
that's the name of the label :)
and also our convention: filepath to directory (or just nlp/tabular/cv), two underscores, and then two underscores and name of file

(that's why I also fixed in this PR one that wasn't like that)